recipes/gw-sklearn-xarray/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "sgw-sklearn-xarray" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mmann1123/sklearn-xarray/archive/refs/tags/v{{ version }}_gw.tar.gz  
+  sha256: cad1c9e62e2700f0ae6299bafcae16aa3716063f784990118d10d97751ebb505
+
+build:
+  skip: True  # [py<37]
```

It looks like this is pure Python and could be noarch:
```suggestion
  noarch: python
```